RFCs/backend/database/tutorial_related_tables/execution_result/execution_result_table.md

```diff
@@ -0,0 +1,21 @@
+# `ExecutionResult` Table
+
+The `ExecutionResult` table contains the execution results. Each of the result is specified by a code snippet and a graph. 
+
+## Mixins
+
+* [`UUIDMixin`](/RFCs/backend/database/mixins.md#UUIDMixin)
+* [`TimeDateMixin`](/RFCs/backend/database/mixins.md#TimeDateMixin)
+
+## Fields
+
+| Field              | Type               | Description |
+| ------------------ | ------------------ | ----------- |
```

Comment:
Should the content in this table be centered in coordination with other tables?

Reply:
Will do.